pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -0,0 +1,111 @@
+package cmd
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	cmdutil "github.com/GoogleCloudPlatform/kubernetes/pkg/kubectl/cmd/util"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/util/errors"
+	"github.com/spf13/cobra"
+
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+	newcmd "github.com/openshift/origin/pkg/generate/app/cmd"
+)
+
+const (
+	newBuildLong = `Create a new build in OpenShift by specifying source code, and/or images.
```

Comment:
...source code, and/or images. -> source code.

(it's either source code or source code + image)